docs/en/advanced_guides/datasets/nuscenes_det.md

```diff
@@ -4,7 +4,11 @@ This page provides specific tutorials about the usage of MMDetection3D for nuSce
 
 ## Before Preparation
 
-You can download nuScenes 3D detection data [HERE](https://www.nuscenes.org/download) and unzip all zip files.
+You can download nuScenes 3D detection `Full dataset (v1.0)` [HERE](https://www.nuscenes.org/download) and unzip all zip files.
```

Comment:
I think we can reopen a new doc `nuscenes_seg.md` for 3D segmentation task

Reply:
We will now unify them into a single `.md`, and our introduction is focused on the dataset, including the different types of tasks on the dataset.